Resources/Prototypes/DeltaV/Roles/Jobs/departments.yml

```
@@ -0,0 +1,9 @@
+- type: department
+  id: Justice
+  description: department-justice-description
+  color: "#701442"
+  roles:
+  - ChiefJustice
+  - Clerk
+  - Prosecutor
+  - Lawyer
```

This isn't Alphabetical and 
```suggestion
  - Lawyer

```